### PR TITLE
Fix bug running multiple triggers

### LINKED ISSUE #387: Single trigger fired multiple times when multiple triggers exist
# Expected Behavior

When I have 3 triggers, I expect each trigger's interceptor to fire once, for each webhook call to the eventlistener.

# Actual Behavior

One trigger is fired 3 times.

# Steps to Reproduce the Problem

1.  Create an eventlistener with multiple triggers (interceptors probably optional)
2.  Trigger the eventlistener
3.  In your event listener pod log you will note logging from sink.go line 147 logging the same /trigger multiple times.

# Additional Info

### PR DESCRIPTION
**Pull request by ncskier**
# Changes
Fixes #387
This PR fixes a pointer bug where an EventListener with `n` triggers
would have the last trigger execute `n` times instead of all `n`
triggers executing once.

This PR also updates the TestHandleEvent function to test multiple
triggers. In the future, it might be worth refactoring the
TestHandleEvent function into a testing table, so we can increase our
test coverage.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: pkg/sink/sink_test.go [hunk 1]
```diff
@@ -131,8 +146,11 @@ func TestHandleEvent(t *testing.T) {
 	if _, err := triggersClient.TektonV1alpha1().TriggerTemplates(namespace).Create(tt); err != nil {
 		t.Fatalf("Error creating TriggerTemplate: %s", err)
 	}
-	if _, err := triggersClient.TektonV1alpha1().TriggerBindings(namespace).Create(tb); err != nil {
-		t.Fatalf("Error creating TriggerBinding: %s", err)
+	// for _, tb := range []*triggersv1.TriggerBinding{tb, tb2, tb3} {
```
Comment (dibyom) [inline pkg/sink/sink_test.go]: Remove?

File: pkg/sink/sink_test.go [hunk 2]
```diff
@@ -195,39 +213,58 @@ func TestHandleEvent(t *testing.T) {
...
-			Type: pipelinev1.PipelineResourceTypeGit,
-			Params: []pipelinev1.ResourceParam{
-				{Name: "url", Value: "testurl"},
-				{Name: "revision", Value: "testrevision"},
-				{Name: "contenttype", Value: "application/json"},
-				{Name: "foo", Value: "bar\t\r\nbaz昨"},
-			},
-		},
-	}
+	// var wantResources []pipelinev1.PipelineResource
```
Comment (dibyom) [inline pkg/sink/sink_test.go]: Remove comment

Comment (wlynch) [inline pkg/sink/sink_test.go]: nit: remove?

**Comment by ncskier** [comment]
Sorry, I forgot to clean up those comments. I created a PR to clean it up here: https://github.com/tektoncd/triggers/pull/389

**Pull request merged**